src/Test/Perf/.gitignore

```diff
@@ -2,5 +2,7 @@ artifacts/
 infra/bin/
 results/
 temp/
+DataBackup*/
+consumptionTempResults.xml
```

Comment:
Right, but we already have a folder for misc temporary files.  Shouldn't these just go there?